src/main/kotlin/assignment/batches.txt

```diff
@@ -1,25 +1,35 @@
-Batch 1:
-1.	Chloe		4283661		CHE
-2.	Jack		5672351		CHE
-3.	Emily		4736471		CIV
-4.	Jacob		3246617		CSE
-5.	Lucy		3248462		CSE
-6.	Sophia		1263612		CSE
+S No.    Name      Roll Number       Branch     Covid Positive
+Batch 1
+1     Chloe 4283661 CHE false
+2     Jack 5672351 CHE false
+3     Emily 4736471 CIV false
```

Comment:
The batches should be of 3 branches at a time and not 3 students. The final output should be same as shown [here](https://github.com/ArpitShukIa/KotlinWorkshop/blob/main/src/main/kotlin/assignment/batches.txt)